src/mcode-pilot/components/TreatmentOptionsOutcomes/TreatmentOptionsOutcomes.jsx

```diff
@@ -3,46 +3,101 @@ import PropTypes from 'prop-types';
 import FontAwesome from 'react-fontawesome';
 import _ from 'lodash';
 
+import { isSame, getCombinations } from '../../utils/arrayCompare';
+
 import BarChart from '../../visualizations/BarChart/BarChart';
+import TableLegend from '../../visualizations/TableLegend/TableLegend';
+import TreatmentsPopover from '../TreatmentsPopover/TreatmentsPopover';
 
 import './TreatmentOptionsOutcomes.css';
 
-const NO_TREATMENT = 'none (actively monitoring)';
+const TREATMENT_NAMES = {
```

Comment:
[GQ1] In TreatmentOptionsOutcomes, why are we directly listing treatment options and side effect names specific to our current data? The list of treatment options and side effects should be ascertained from the data returned from the treatment options service - not hard coded! We had this in the treatment options viz we did originally.